Marlin/src/lcd/menu/menu_item.h

```diff
@@ -343,7 +343,7 @@ class MenuItem_bool : public MenuEditItemBase {
 #define PSTRING_ITEM(LABEL, V...)                     PSTRING_ITEM_P(GET_TEXT(LABEL), ##V)
 
 #define STATIC_ITEM(LABEL,   V...)                     STATIC_ITEM_P(GET_TEXT(LABEL), ##V)
-#define STATIC_ITEM_N(LABEL, V...)                   STATIC_ITEM_N_P(GET_TEXT(LABEL), ##V)
+#define STATIC_ITEM_N(LABEL, N, V...)                   STATIC_ITEM_N_P(GET_TEXT(LABEL), N, ##V)
```

Comment:
Note that the only benefit here is that the compiler will insist on at least 3 parameters. Otherwise the code functions perfectly fine as it is.